### ISSUE TITLE
Issue for tracking issues

### ISSUE THREAD

**Issue by alexomics** [LooseLab/readfish#174]
Tracking issues for tidying up this repo.

Branches/issues:
 - [`Adoni5-patch-1`][Adoni5-patch-1] patch typo in README.md
 - [`barcoding`][barcoding] Adds barcoding compatibility 
 - [`caller_refactor`][caller_refactor] Adds CPU basecalling
 - [`dev_documentation`][dev_documentation] documentation branch
 - [`guppy_6`][guppy_6] GH-170
 - [`issue101`][issue101] GH-101
 - [`issue109`][issue109] GH-109 (Handle closing down readfish and errors better)
 - ~[`issue131`][issue131]~ GH-131 Closed in GH-132
 - [`issue137`][issue137] GH-137 (Assess sequence length prior to sending unblocks)
 - [`issue15`][issue15] GH-15 (Move to later guppy versions)
 - ~[`issue153`][issue153]~ GH-153 (cannot write to minknow folder) closed in GH-175
 - ~[`issue155`][issue155]~ GH-155 (MinKNOW messaging fix) closed in GH-164
 - ~[`issue167`][issue167]~  GH-167 (`lru_cache` shenanigans) closed in GH-176
 - [`issue_134`][issue_134] GH-134 (BOSS-RUNS branch)

Branches from here will be merged into [`dev_staging`][dev_staging] and struck from the list. Items crossed out may not exist anymore. 

 [Adoni5-patch-1]: https://github.com/LooseLab/readfish/tree/Adoni5-patch-1
 [barcoding]: https://github.com/LooseLab/readfish/tree/barcoding
 [caller_refactor]: https://github.com/LooseLab/readfish/tree/caller_refactor
 [dev_documentation]: https://github.com/LooseLab/readfish/tree/dev_documentation
 [guppy_6]: https://github.com/LooseLab/readfish/tree/guppy_6
 [issue101]: https://github.com/LooseLab/readfish/tree/issue101
 [issue109]: https://github.com/LooseLab/readfish/tree/issue109
 [issue131]: https://github.com/LooseLab/readfish/tree/issue131
 [issue137]: https://github.com/LooseLab/readfish/tree/issue137
 [issue15]: https://github.com/LooseLab/readfish/tree/issue15
 [issue153]: https://github.com/LooseLab/readfish/tree/issue153
 [issue155]: https://github.com/LooseLab/readfish/tree/issue155
 [issue167]: https://github.com/LooseLab/readfish/tree/issue167
 [issue_134]: https://github.com/LooseLab/readfish/tree/issue_134
 [dev_staging]: https://github.com/LooseLab/readfish/tree/dev_staging
